Code/Tools/ProjectManager/Source/FormLineEditTagsWidget.cpp

```diff
@@ -0,0 +1,195 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzQtComponents/Components/Widgets/LineEdit.h>
+#include <AzQtComponents/Components/StyledLineEdit.h>
+
+
+#include <FormLineEditTagsWidget.h>
+#include <FormLineEditWidget.h>
+
+#include <QAbstractItemView>
+#include <QCheckBox>
+#include <QCompleter>
+#include <QFile>
+#include <QFrame>
+#include <QHBoxLayout>
+#include <QKeyEvent>
+#include <QLabel>
+#include <QLineEdit>
+#include <QPushButton>
+#include <QSpacerItem>
+#include <QStyle>
+#include <QValidator>
+#include <QVBoxLayout>
+
+namespace O3DE::ProjectManager
+{
+
+    void FormLineEditTagsWidget::setupCompletionTags()
+    {
+        QFile completionTagFile(":/ProjectManager/text/ProjectManagerCompletionTags.txt");
```

Comment:
We need to make sure this file ends up in the installer build (SDK output) or is optional - have you tested this where the file is absent?  I'm OK with this change going in a separate PR, but just wanted to make sure it won't break for the installer version.

Reply:
If you want to add this to the SDK layout, you can us the [ly_install_files](https://github.com/o3de/o3de/blob/development/cmake/Install.cmake#L116-L125) or [ly_install_directory](https://github.com/o3de/o3de/blob/development/cmake/Install.cmake#L37-L52) command to copy this file to the SDK layout.

You can then build the `install` target(called INSTALL) in visual studio to create an SDK layout and validate that the file is there

Reply:
I missed that this file is part of the .qrc so it should be included automatically with the other Qt resource files.